sql/updates/world/2016_01_14_99_world.sql

```diff
@@ -0,0 +1,307 @@
+UPDATE `creature_template` SET  `minlevel`=80,`maxlevel`=80, `ScriptName`='npc_frozen_core' WHERE `entry`=25865; -- Frozen Core
+UPDATE `creature_template` SET `ScriptName`='npc_ahune_bunny' WHERE `entry`=25745;
+UPDATE `creature_template` SET `ScriptName`='npc_earthen_ring_flamecaller' WHERE `entry`=25754;
+UPDATE `creature_template` SET `unit_flags`=33554432, `MovementType`=2 WHERE `entry` IN (25964,25965,25966); -- Shaman beam bunny
+UPDATE `creature_template` SET `unit_flags`=33554432 WHERE `entry`=26239; -- Ghost of Ahune
+UPDATE `creature_template` SET `flags_extra`=128 WHERE `entry`=25985; -- Ahune Ice Spear Bunny
+UPDATE `gameobject_template` SET `ScriptName`='go_ahune_ice_stone' WHERE `entry`=187882;
+UPDATE `creature` SET `orientation`=2.408554 WHERE `guid`=202734; -- Luma
+UPDATE `creature` SET `orientation`=3.804818 WHERE `guid`=202737; -- Flamecaller
+UPDATE `creature_template` SET `HealthModifier`=94.5, `unit_flags`=33554688 WHERE `entry`=25865;
+UPDATE `creature_template` SET `HealthModifier`=18.8 WHERE `entry`=25755;
+UPDATE `creature_template` SET `HealthModifier`=3.538 WHERE `entry`=25756;
+UPDATE `creature_template` SET `HealthModifier`=1.5 WHERE `entry`=25757;
+UPDATE `creature_template` SET `HealthModifier`=4 WHERE `entry`=40446;
+UPDATE `creature_template` SET `InhabitType`=7 WHERE `entry` IN (25964,25965,25966,26190);
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN
+('spell_ahune_synch_health',
+'spell_ice_spear_control_aura',
+'spell_slippery_floor_periodic',
+'spell_summon_ice_spear_delayer',
+'spell_summoning_rhyme_aura',
+'spell_ahune_spanky_hands',
+'spell_ahune_minion_despawner',
+'spell_ice_spear_target_picker',
+'spell_ice_bombardment_dest_picker');
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(46430, 'spell_ahune_synch_health'),
+(46371, 'spell_ice_spear_control_aura'),
+(46320, 'spell_slippery_floor_periodic'),
+(46878, 'spell_summon_ice_spear_delayer'),
+(45926, 'spell_summoning_rhyme_aura'),
+(46146, 'spell_ahune_spanky_hands'),
+(46843, 'spell_ahune_minion_despawner'),
+(46372, 'spell_ice_spear_target_picker'),
+(46398, 'spell_ice_bombardment_dest_picker');
+
+DELETE FROM `creature_text` WHERE `entry` IN (25745,25754,25697,40446);
+INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(25745,0,0,'The Earthen Ring\'s Assault Begins.',41,0,100,0,0,0,24930,1,'Ahune Bunny- EMOTE_EARTHEN_ASSAULT'),
+(25754,0,0,'Ahune Retreats.  His defenses diminish.',41,0,100,0,0,0,24931,1,'Earthen Ring Flamecaller - EMOTE_RETREAT'),
+(25754,1,0,'Ahune will soon resurface.',41,0,100,0,0,0,24932,1,'Earthen Ring Flamecaller - EMOTE_RESURFACE'),
+(40446,0,0,'How DARE you! You will not stop the coming of Lord Ahune!',14,0,100,0,0,0,40437,0,'Skar\'this the Summoner'),
+(25697,0,0,'The Ice Stone has melted!',14,0,100,0,0,0,24895,0,'Luma Skymother - SAY_PLAYER_1'),
```

Comment:
https://github.com/TrinityCore/TrinityCore/commit/8f71950c910ebf76b1335d710f2ec584aaa58fca#commitcomment-15530581 from Killyana:
"Blizzard mistake we will not reproduce their bugs."
Might be meant only for specific issues, though.